examples/team_with_custom_roles/team.yaml

```diff
@@ -46,7 +53,9 @@ metadata:
   name: allow-product-manager-tag
 spec:
   forProvider:
-    description: "Allow Product Managers to toggle flags and change targeting for flags wtih the product-manager tag"
+    description: |
+      Allow Product Managers to toggle flags and change targeting for flags with the product-manager tag.
+      This role uses the `productManagerProjAttr` role attribute so that projects can be assigned at the team level.
```

Comment:
```suggestion
      This role leverages role scope with the `productManagerProjAttr` role attribute so that projects can be assigned at the team level.
```